cypress/components/menu/menu.cy.tsx

```diff
@@ -569,25 +177,27 @@ context("Testing Menu component", () => {
     );
 
     it.each([
-      ["white", "first", "rgb(255, 255, 255)"],
-      ["light", "fifth", "rgb(230, 235, 237)"],
-      ["dark", "ninth", "rgb(0, 50, 76)"],
-      ["black", "thirteenth", COLOR.BLACK],
-    ])("should verify Menu is %s menuType", (menuType, menuNumber, color) => {
-      CypressMountWithProviders(<MenuComponent />);
-
-      menuItem()
-        .eq(positionOfElement(menuNumber))
-        .children()
-        .should("have.css", "background-color", color);
-    });
+      ["rgb(255, 255, 255)", "first"],
+      ["rgb(230, 235, 237)", "fifth"],
+      ["rgb(0, 50, 76)", "ninth"],
+      [COLOR.BLACK, "thirteenth"],
+    ] as [string, "first" | "fifth" | "ninth" | "thirteenth"][])(
+      "should verify Menu is %s menuType",
+      (color, menuNumber) => {
+        CypressMountWithProviders(<MenuComponent />);
+        menuItem()
+          .eq(positionOfElement(menuNumber))
+          .children()
+          .should("have.css", "background-color", color);
+      }
+    );
 
     it.each([
       ["white", "rgba(0, 0, 0, 0.9)"],
       ["light", "rgba(0, 0, 0, 0.9)"],
       ["dark", "rgb(255, 255, 255)"],
       ["black", "rgb(255, 255, 255)"],
-    ])(
+    ] as [MenuProps["menuType"], string][])(
       "should verify icons have the correct color when menuType is %s",
       (menuType, color) => {
```

Comment:
Despite my opinion on it, `menuType` has been left in here, so if the majority of reviewers believe we should remove unused params, this should be deleted. Alternatively, if we agree to have them in, this should remain. I just believe whatever the approach, it should be consistent in every test 👍